azure-pipelines.yml

```diff
@@ -1,4 +1,26 @@
 jobs:
+- job:
```

Comment:
These changes need to be in a different PR.

Reply:
The related commit has be cherry-picked to develop branch .